CHANGELOG.md

```diff
@@ -1,9 +1,24 @@
-## 3.1.5 (TBD, 2024)
+# 3.1.5 (July 18, 2024)
+
 ###
-- Supported Controller version: **UserConnect-TBD**
+- Supported Controller version: **UserConnect-7.1.4105**
 - Supported Terraform version: **v1.x**
 
-### Deprecations
+## Bug Fixes:
+
+1. Fixed issue in Firenet and FQDN Gateway for Azure where we no longer require special handling of ``fqdn_lan_interface``  and ``lan_interface``.
```

Comment:
should we list the specific resources that are affected? Since these affected attributes listed do not have a frame of reference.